pkg/api/types.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	docker "github.com/fsouza/go-dockerclient"
+	"time"
+)
+
+// OpenSCAPStatus is the status of openscap scan
+type OpenSCAPStatus string
+
+const (
+	StatusNotRequested OpenSCAPStatus = "NotRequested"
+	StatusSuccess      OpenSCAPStatus = "Success"
+	StatusError        OpenSCAPStatus = "Error"
+)
+
+type OpenSCAPMetadata struct {
+	Status           OpenSCAPStatus // Status of the OpenSCAP scan report
+	ErrorMessage     string         // Error message from the openscap
+	ContentTimeStamp string         // Timestamp for this data
```

Comment:
TBH it is string because it was string before [1], I didn't go into this that much when I moved these pieces of code around. I agree that it will be nice to have it as Time but I think that changing this should be done in a different PR because the issue is different than the one this PR is about.

[1]https://github.com/openshift/image-inspector/pull/39/files#diff-f5d24f0dc00fe1c3eeda8d22b2e523ebL20